### ISSUE TITLE
gphoto compatability

### ISSUE THREAD

**Issue by geekscrapy** [drumminhands/drumminhands_photobooth#6]
Hi can this be used with gphoto? I don't have the rPi camera but I've got a DSLR!

Thanks!

**Comment by thebluehill** [comment]
That would be very nice! I'm also interested to use a DSLR with gphoto2.
Is there a way it insert gphoto?

Thanks

**Comment by Jimbles** [comment]
yeah There is! I had intended to do this one my version but I didnt want to leave the DSLR unattended during the evening when things can get rowdy!

Most of the stuff stays the same, but the Overlay and Capture code would change [this](https://superuser.com/questions/870768/can-i-use-my-dslr-as-webcam-through-gphoto2-v4l2loopback) looks good for the overlay (im not sure about overlaying text though) and you can then use the standard gphoto capture commands for the photos
